### ISSUE TITLE
simplify fails on compareCluster result when ont passed as variable

### ISSUE THREAD

**Issue by malcook** [GuangchuangYu/clusterProfiler#72]
Your function `simplify`, as written, fails to retrieve the GO ontology when compareCluster passes a variable as value of ont.

For example, taking your working example from https://github.com/GuangchuangYu/clusterProfiler/issues/28#issuecomment-254518131

but changing it to pass `ont` as a variable, as follows

```
data(gcSample)
vont<-'MF'
xx = compareCluster(gcSample, fun='enrichGO', ont=vont, OrgDb='org.Hs.eg.db')
y <- simplify(xx, measure='Wang', semData=NULL) 
```

produces

```
Error in match.arg(ont, c("BP", "CC", "MF")) : 
  'arg' should be one of “BP”, “CC”, “MF”
```

```
> rvcheck::check_bioc("clusterProfiler")
package is up-to-date release version
$package
[1] "clusterProfiler"

$installed_version
[1] "3.2.6"

$latest_version
[1] "3.2.6"

$up_to_date
[1] TRUE
```

**Comment by GuangchuangYu** [comment]
thanks for your feedback. It will be fixed on clusterProfiler >=3.2.7.

**Comment by malcook** [comment]
Excellent and thanks.  I do confirm it now works as hoped when using:
`  
install_github("GuangchuangYu/clusterProfiler")
`
